### PR TITLE
resource-directory: refactoring

### PR DESCRIPTION
**Pull request by Danielius1922**

### CODE REVIEW

File: resource-directory/service/filterBitmask.go [hunk 1]
```diff
@@ -44,13 +44,12 @@ func filterPendingCommandToBitmask(f pb.GetPendingCommandsRequest_Command) filte
 }
 
 func filterPendingsCommandsToBitmask(commandFilter []pb.GetPendingCommandsRequest_Command) filterBitmask {
```
Comment (jkralik) [inline resource-directory/service/filterBitmask.go]: maybe you can use filterBitmask from https://github.com/plgd-dev/cloud/blob/v2/grpc-gateway/subscription/filterBitmask.go

**Comment by jkralik** [comment]
Did you update the c2c-gw docs for change RD to grpc-gw?

File: resource-directory/service/filterBitmask.go (hunk 1 above)
Comment (Danielius1922) [reply resource-directory/service/filterBitmask.go]: Yes, it's the same. Switched to use the grpc-gw/subscription code.

**Comment by Danielius1922** [comment]
> Did you update the c2c-gw docs for change RD to grpc-gw?

Docs updated.

File: authorization/client/ownerCache.go [hunk 2]
```diff
@@ -109,6 +109,9 @@ func (d *ownerSubject) syncDevicesLocked(ctx context.Context, owner string, cach
 	return added, removed, nil
 }
 
+// ErrFunc reports errors
+type ErrFunc func(err error)
```
Comment (jkralik) [inline authorization/client/ownerCache.go]: change `type ErrFunc func(err error)` to alias `type ErrFunc = func(err error)` we don't need to be strict

**Review by jkralik — APPROVED** [review]

**Pull request merged**